nodejs/device/src/index.ts

```diff
@@ -0,0 +1,318 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
global comment: I know that Tony is push using prettier to format code in the node repo.  We should do that here, following the same conventions.

Reply:
Agreed